docs/intro.rst

```diff
@@ -0,0 +1,4 @@
+Introduction
```

Comment:
Oh yeah, missed that one. The intention was for the intro to go in `index` before the table of contents. I'll remove this.